app/controllers/apicasso/crud_controller.rb

```diff
@@ -129,7 +129,9 @@ def set_records
     # Selects a fieldset that should be returned, instead of all fields
     # from records.
     def select_fields
-      @records = @records.select(*params[:select].split(','))
+      params[:select].split(',').each do |p|
```

Comment:
`params[:select].split` is present on `parsed_select`, use it to avoid code repetition.